lxml.h

```diff
@@ -413,8 +477,8 @@ static void node_out(FILE* file, XMLNode* node, int indent, int times)
         XMLNode* child = node->children.data[i];
 
         if (times > 0)
-            fprintf(file, "%0*s", indent * times, " ");
-        
+            fprintf(file, "%*s", indent * times, " ");
```

Comment:
`GCC`/`GNU` was warning about this behaviour? I wasn't sure what `%0*s` is supposed to do?